services/obstap/values-usdfdev.yaml

```diff
@@ -0,0 +1,16 @@
+cadc-tap-postgres:
+  fullnameOverride: "obstap"
+
+  imagePullSecrets:
+    - name: "pull-secret"
+  ingress:
+    host: "rubin-data-dev.slac.stanford.edu"
+  vaultSecretsPath: "secret/rubin/usdf-staffrsp-dev/tap"
+
+  config:
+    gcsBucket: "async-results.lsst.codes"
+    gcsBucketUrl: "http://async-results.lsst.codes"
+
+pull-secret:
+  enabled: true
+  path: secret/rubin/usdf-staffrsp-dev/pull-secret
```

Comment:
obstap is currently obsolete. It will eventually be resurrected, but in the meantime this file should be deleted since when it comes back it will be in a different form.